### PR TITLE
Added possibility to disable RHMP

### PR DESCRIPTION
**Pull request by kmigielek**
gracefully removes k8s resources which are related to rhmp when it gets disabled

### CODE REVIEW

File: controllers/resources/service/services.go [hunk 1]
```diff
@@ -38,19 +38,23 @@ var (
...
+	} else {
+		notExpected = append(notExpected, prometheusService)
 	}
 
-	if s := GetUsageService(instance); s != nil {
-		services = append(services, s)
+	usageService := GetUsageService(instance)
+	if instance.Spec.UsageEnabled {
+		expected = append(expected, usageService)
+		return
```
Comment (kacperkrzyzak) [inline controllers/resources/service/services.go:54]: Add usage Service to notExpected for disabled Usage

**Review by kacperkrzyzak — CHANGES_REQUESTED** [review]

File: controllers/resources/service/services.go (hunk 1 above)
Comment (kmigielek) [reply controllers/resources/service/services.go:54]: its added to notExpected before return

**Review by arturobrzut — APPROVED** [review]

**Comment by arturobrzut** [comment]
open new PR 
kacper on vacation

**Pull request closed without merging**